supporting_scripts/hyperion/consistency-check-benchmark/merge_request.py

```
@@ -0,0 +1,293 @@
+                cwd=pecv_bench_dir,
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+            logging.info(f"Checked out existing remote branch '{branch_name}'.")
+            # Re-stage results after checkout (checkout cleared the index)
+            _stage_results_and_runs(pecv_bench_dir, approach_id)
+        else:
+            # Base the new branch on origin/{PECV_BENCH_BRANCH} so the PR diff
+            # only shows this approach's results, not other locally committed approaches.
+            subprocess.run(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)